src/js/pages/jobs/JobDetailPage.js

```diff
@@ -143,21 +149,34 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
   }
 
   getActionButtons() {
-    let dropdownItems = [
-      {
-        className: 'hidden',
-        html: 'More',
-        id: JobActionItem.MORE
-      },
-      {
+    let job = MetronomeStore.getJob(this.props.params.id);
+    let dropdownItems = [];
+    let [schedule] = job.getSchedules();
+
+    dropdownItems.push({
+      className: 'hidden',
+      html: 'More',
+      id: JobActionItem.MORE
+    });
+
+    if (schedule != null && schedule.enabled) {
```

Comment:
You arrange and shorten logically (I mean the maths logic).
If you don't do it, you unnecessarily evaluate `schedule != null` twice!